test/naming.dart

```diff
@@ -19,19 +19,27 @@ const String platformBrowser = 'browser';
 const String platformMock = 'mock';
 const String platformVM = 'vm';
 
-/// Integration test group names.
-const String integrationHttpBrowser = '[integration] HTTP ($platformBrowser)';
-const String integrationHttpMock = '[integration] HTTP ($platformMock)';
-const String integrationHttpVM = '[integration] HTTP ($platformVM)';
-const String integrationWebSocketBrowser =
-    '[integration] WS ($platformBrowser)';
-const String integrationWebSocketMock = '[integration] WS ($platformMock)';
-const String integrationWebSocketVM = '[integration] WS ($platformVM)';
+/// Test types.
+const String testTypeIntegration = 'integration';
+const String testTypeUnit = 'unit';
 
-/// Unit test group names.
-const String unitHttpBrowser = '[unit] HTTP ($platformBrowser)';
-const String unitHttpMock = '[unit] HTTP ($platformMock)';
-const String unitHttpVM = '[unit] HTTP ($platformVM)';
-const String unitWebSocketBrowser = '[unit] WS ($platformBrowser)';
-const String unitWebSocketMock = '[unit] WS ($platformMock)';
-const String unitWebSocketVM = '[unit] WS ($platformVM)';
+/// Topics
+const String topicHttp = 'HTTP';
+const String topicMocks = 'Mocks';
+const String topicPlatformAdapter = 'Platform Adapter';
+const String topicWebSocket = 'WS';
+
+class Naming {
+  String platform;
+  String testType;
+  String topic;
+
+  @override
```

Comment:
It's actually never necessary, the benefit you get from it is if you include it on something that doesn't actually override something from a parent class. So if you have a typo or if you refactor something in a parent class and forget to change the child class, the compiler can warn you.